packages/openapi-typescript/test/schema-object.test.ts

```diff
@@ -49,12 +49,6 @@ describe("Schema Object", () => {
   status?: "complete" | "incomplete";
 }`);
       });
-
-      test("nullable", () => {
```

Comment:
Removed the disparate nullable tests and grouped them together (which helped identify several missing test cases)